docs/man/dune.1.md

```diff
@@ -0,0 +1,291 @@
+---
+title: DUNE
+section: 1
+header: User Manual
+footer: DUNE 0.0.1
+date: April 04, 2023
+---
+# NAME
+DUNE - Docker Utilities for Node Execution.
+
+# SYNOPSIS
+
+`dune <option> [<arguments>] [-- <commands>]`
+
+# DESCRIPTION
+
+**DUNE** is a utility to interact with the EOS blockchain system installed in a Docker container. 
+The main purpose of this utility is to provide all features needed for management of EOS in one
```

Comment:
for management of EOS `blockchain`

Reply:
I hope this description is better. I took it from the readme.md